8/SiloHost/src/SiloConfigurationHelper.cs

```diff
@@ -0,0 +1,127 @@
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.NetworkInformation;
+using System.Net.Sockets;
+using System.Text.Json;
+using Amazon.Util;
+using Orleans.Configuration;
+using OrleansDashboard;
+
+namespace SiloHost
+{
+    public class SiloConfigurationHelper:ISiloConfigurationHelper
```

Comment:
This also acts as a service, let's leverage IoC to pass its parameters to the constructor.